src/parsy/__init__.py

```diff
@@ -194,7 +213,7 @@ def alt_parser(stream, index):
 
     return alt_parser
 
-def seq(*parsers):
+def seq(*parsers, f=None):
```

Comment:
Well, the difference is that you almost certainly wouldn't pass arguments to `map`/`bind` by name (and most likely you wouldn't even know what they're called), whereas in this case it's keyword-only, so I wanted to use a shorter name. I do not insist though. Let's wait for @jneen to decide.